react/features/overlay/actions.ts

```diff
@@ -26,37 +28,15 @@ export function mediaPermissionPromptVisibilityChanged(isVisible: boolean, brows
 }
 
 /**
- * The action indicates that an unrecoverable error has occurred and the reload
- * screen will be displayed or hidden.
+ * Opens {@link PageReloadDialog}.
  *
- * @param {Object} fatalError - A critical error which was not claimed by any
- * feature for error recovery (the recoverable flag was not set). If
- * {@code undefined} then any fatal error currently stored will be discarded.
- * @returns {{
- *     type: SET_FATAL_ERROR,
- *     fatalError: ?Error
- * }}
+ * @protected
+ * @returns {Action}
  */
-export function setFatalError(fatalError?: Object) {
-    return {
-        type: SET_FATAL_ERROR,
-        fatalError
-    };
-}
+export function fatalError(is?: boolean) {
+    if(is) {
+        openDialog(PageReloadDialog)
```

Comment:
you need to dispatch this

Reply:
We have other actions that use openDialog and we dispatch them once. 
For ex. openLoginDialog.